sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -93,7 +93,7 @@ export interface CreateRsaKeyOptions extends CreateKeyOptions {
 // @public
 export class CryptographyClient {
     constructor(key: string | KeyVaultKey, credential: TokenCredential, pipelineOptions?: CryptographyClientOptions);
-    constructor(key: JsonWebKey_2);
+    constructor(key: JsonWebKey);
```

Comment:
The removal of `dom` from the list of TS libraries cleaned up the api view since there is no longer naming conflict 😊 However, I wonder why `dom` was being included in the first place?

Reply:
Oh man you and me both 😄 https://github.com/Azure/azure-sdk-for-js/issues/31268 was created back when we were moving to ESM to track this - looks like based on your PR feedback! 

I am more than happy to remove `dom` here as long as its not breaking for our customers. I get that KeyVault is of course not actually supported in the browser but would any builds / test runs start breaking for customers who maybe disable web security for tests? Maybe you happen to know

Reply:
Ah nice 😅

I don't believe this would affect external customers because including `dom` only affects the typing context for the source code and what we ship to our users is the compiled js and the d.ts files. Let's merge it as is and we can reconsider if anything gets affected.